Remove IE polyfills and special casing (#2947)
transloadit/uppy: README.md

  <img src="https://saucelabs.com/browser-matrix/transloadit-uppy.svg" alt="Sauce Test Status"/>
</a>

We aim to support recent versions of Safari, Edge, Chrome, Firefox and Opera.

> We still run end-to-end tests with IE11, but we are not actively supporting it or fixing visual / minor issues.

Reviewer: Does something still need to happen in order to run e2e tests against IE11 if we just removed the polyfills?

Reviewer: Seems we just removed e2e tests for IE11 so this should change

### Polyfills

> Here's a list of polyfills you'll need to include to make Uppy work in older browser (e.g., IE11):

Reviewer: I usually prefer things as "such as" over "e.g." as it reads more pleasant and is less pretentious.

```suggestion
Here's a list of polyfills you'll need to include to make Uppy work in older browsers, such as IE11:
```

- [abortcontroller-polyfill](https://github.com/mo/abortcontroller-polyfill)
- [es6-promise](https://github.com/stefanpenner/es6-promise)
- [math-log2](https://github.com/sindresorhus/math-log2)
- [md-gum-polyfill](https://github.com/mozdevs/mediaDevices-getUserMedia-polyfill)
- [resize-observer-polyfill](https://github.com/que-etc/resize-observer-polyfill)
- [url-polyfill](https://github.com/lifaon74/url-polyfill)
- [whatwg-fetch](https://github.com/github/fetch)

If you're using a bundler, you need import them before Uppy:

```js

> import 'es6-promise/auto';

Reviewer: Uppy is not going to move to ESM in `2.0` so probably best to keep these examples with `require`.

Author: I disagree, I think we should start to push for ESM actually. ESM can import both ESM and CJS – except in Deno, because they don't support CJS at all – us internally using CJS for the time being shouldn't refrain our users to switch to ESM (we really want that to happen before 3.0 actually).

Reviewer: Fair enough! But then I would argue for consistency, so _every_ README and documentation example. Perhaps a bit too much for this PR?

Author: Agreed! I've assigned https://github.com/transloadit/uppy/issues/2951 to myself.
